app/drivers/tools/repair/java/GRT5.py

```diff
@@ -0,0 +1,137 @@
+import os
+import json
+from pathlib import Path
+
+from app.drivers.tools.repair.AbstractRepairTool import AbstractRepairTool
+
+
+class GRT5(AbstractRepairTool):
+    def __init__(self):
+        self.name = os.path.basename(__file__)[:-3].lower()
+        super().__init__(self.name)
+        #self.image_name = "grt5-dev"
+        self.image_name = "xmcp/grt5:231004.1"
+        self.hash_digest = "sha256:e900657c34545bd5320aacaef6c299f80b782449c86ae447c66984837af63b1e"

```

Comment:
Hi... Just a small thing in relation to this digest, earlier the code in Cerberus was incorrect and was using the local ID of the image. This has been fixed, and I was looking at some of the images. Just to gently inform about this fix, sorry 😟

Reply:
@AuQWQuA can you quickly check this small change, and help us to close this PR?

Reply:
sorry I just noticed it and I fixed it

Reply:
Thank you!